visualizations/frontend/charts/line_chart.html

```diff
@@ -3,458 +3,544 @@
 </template>
 
 <figure>
-{
-    {% set line_colors = var('line_colors') %}
-    {% set fill_colors = var('fill_colors') %}
-    {% set font_family = var('font_family') %}
-    {% set font_color = var('font_color') %}
-    {% set font_size = var('font_size') %}
-    {% set tickfont_family = var('tickfont_family') %}
-    {% set tickfont_color = var('tickfont_color') %}
-    {% set tickfont_size = var('tickfont_size') %}
-    {% set hover_label_bg = var('hover_label_bg') %}
```

Comment:
Aren't these required? These variables are referenced in the code.

Reply:
Just fixed this referenced code.